release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -114,6 +114,13 @@ You can customize the MTU for a cluster by specifying the networking.clusterNetw
 
 For more information, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installation-aws-cluster-network-mtu_installing-aws-localzone[Customizing the cluster network MTU].
 
+[id="ocp-4-15-installing-aws-outposts"]
+==== Installing a cluster on AWS with compute nodes on AWS Outposts
+
+In {product-title} version 4.14, you could install a cluster on AWS with compute nodes running in AWS Outposts as a Technology Preview. In {product-title} 4.15, you can install a cluster on AWS into an existing VPC and provision compute nodes on AWS Outposts as a postinstallation configuration task.
+
+For more information, see xref:../installing/installing_aws/installing-aws-vpc.adoc[Installing a cluster on AWS into an existing VPC].
```

Comment:
@bscott-rh, this should link to _Extending an AWS VPC cluster into an AWS Outpost_?

Reply:
There will be a second link to the Extending an AWS VPC cluster into an AWS Outpost page after https://github.com/openshift/openshift-docs/pull/71263 merges

Reply:
https://github.com/openshift/openshift-docs/pull/72145